From 219b6d8ecc6342fd1065c459f965e9116d214c33 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:53 -0400
Subject: [redhat] kabi: Adding symbol yield (kernel/sched/core.c)

Message-id: <7cf1a179f893340f5ef1b9aac228223e61c23301.1410510773.git.poros@redhat.com>
Patchwork-id: 96004
O-Subject: [PATCH RHEL7.1 233/233] kabi: Adding symbol yield (kernel/sched/core.c)
Bugzilla: 1062361 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1062361,1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    This function is used to voluntarily pass processor control to some other kernel thread.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/yield
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/yield
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/yield

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/yield b/redhat/kabi/kabi-module/kabi_ppc64/yield
new file mode 100644
index 0000000..4f850f8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/yield
@@ -0,0 +1,2 @@
+#1-
+0x760a0f4f yield vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/yield b/redhat/kabi/kabi-module/kabi_s390x/yield
new file mode 100644
index 0000000..4f850f8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/yield
@@ -0,0 +1,2 @@
+#1-
+0x760a0f4f yield vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/yield b/redhat/kabi/kabi-module/kabi_x86_64/yield
new file mode 100644
index 0000000..4f850f8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/yield
@@ -0,0 +1,2 @@
+#1-
+0x760a0f4f yield vmlinux EXPORT_SYMBOL
-- 
1.7.1